float64 value
string group_name
string data_name
---
bool resp
